site.go

```diff
@@ -17,8 +23,9 @@ type envConfig struct {
 	Port     int    `envconfig:"PORT" default:"8080" required:"true"`
 }
 
+var env envConfig
```

Comment:
I hoisted this into a global so it was easy to get to `DataPath` outside of main. Happy to encapsulate more if you want.